src/js/components/reviewData/ReviewDataContent.jsx

```diff
@@ -175,15 +175,16 @@ export default class ReviewDataContent extends React.Component {
 
         let certifyButtonText = "You do not have permissions to certify";
         let notifyButtonText = "Notify Another User that the Submission is Ready for Certification";
-        let buttonClass = "btn-disabled";
+        let buttonClass = " btn-disabled";
         let buttonAction = "";
         let monthlySubmissionError = null;
+
         if (this.props.data.publish_status == "published") {
             certifyButtonText = "Submission has already been certified";
             buttonClass = " btn-disabled";
```

Comment:
In all of these "if" statements other than the last one, you don't have to change the buttonAction or buttonClass because you're just changing it to what's already there. The only time it has to be changed is if we stop it from being disabled and allow the user to click on it.